.circleci/config.yml

```diff
@@ -19,10 +19,10 @@ commands:
       - run:
           name: Docker login
           command: |
-            echo "Login in $DOCKER_REGISTRY with user $DOCKER_USER"
-            echo $DOCKER_PASS | docker login -u $DOCKER_USER --password-stdin $DOCKER_REGISTRY
-            echo "Login in $DOCKER_REGISTRY_2 with user $DOCKER_USER_RW"
-            echo $DOCKER_PASSWORD_RW | docker login -u $DOCKER_USER_RW --password-stdin $DOCKER_REGISTRY_2
+            echo "Login in cloudsmith"
+            echo $CLOUDSMITH_PASS_RW | docker login docker.consensys.net -u $CLOUDSMITH_USER_RW --password-stdin
```

Comment:
Improved how we used to work with project envs and use contexts which appear to be the best approach working with shared credentials across projects